messenger/handler_results.rst

```diff
@@ -36,7 +31,12 @@ buses are central pieces of the application. Read Martin Fowler's
 As queries are usually synchronous and expected to be handled once,
 getting the result from the handler is a common need.
 
-To avoid boilerplate code, you can leverage the ``HandleTrait`` in any class
+A :class:`Symfony\\Component\\Messenger\\HandleTrait` exists in order to ease
+leveraging a Messenger bus for this synchronous needs.
+The :method:`Symfony\\Component\\Messenger\\HandleTrait::handle` method ensures
+there is exactly one handler registered and returns its result.
```

Comment:
... exists to make it easier to get the result of the handler when processing asynchronously. It also em Aires exactly one handler is registered. The HandleTrait can be used in any class that has a messageBus property:

Let’s use this (and then no text needed below - go straight to code)

Reply:
@weaverryan sorry i do not understand your comment

is it the rewording sentence you propose?
Maybe can you do it? i only wanted to point that this part is more usefull at another place of the doc :)

thank you

Reply:
Wow, sorry - my comment was very unclear - reviewing from my phone :/. I just edited my comment. What I was thinking (and it’s already close now) is, starting with the word “exists”, to replace the rest with my text above. Then, the next thing after the text would be the code itself.

Moving the text I think makes sense. So now I just want to reword this section.